internal-employee-directory/backend/sample.js

```diff
@@ -0,0 +1,109 @@
+const data = [
```

Comment:
I would seed it w/ random data instead of our legit names etc etc

Reply:
ok will do that